src/lib/ops/organizationOps.js

```diff
@@ -200,7 +202,9 @@ var organizationOps = {
 
     const tasks = [
       (job, next) => {
-        const { id, name, description, user } = params
+        const id = params.id || uuid()
```

Comment:
This was the https://github.com/nearform/labs-authorization/issues/324 issue about. In `teams` and `users` the ids are generated if not present. Why would the `organisation` be different?